src/config/index.ts

```diff
@@ -24,30 +25,22 @@ export const STATE_FILE_NAME = "state.json";
 export const STATE_FILE_PATH = `${REPOS_DIRECTORY_PATH}/${STATE_FILE_NAME}`;
 
 export const HEADERS = { 'headers': { 'Authorization': `Bearer ${GITHUB_KEY}` } };
+export const API_CLIENT = createOAuthApiClient(GITHUB_KEY);
 
 export const ORG_DATA: OrgData = {
     "repos": { "list": [], "details": {} },
     "members": { "list": [], "details": {} },
     "teams": { "list": [], "details": {} }
 };
 
-export const MEMBERS_PER_TEAM_KEY = "members_per_team";
```

Comment:
This is part of another story. Member per teams and repos per teams will be covered on `NTRNL-254` and `NTRNL-255`

Reply:
This change was part of the last commit which was the deletions, so I can remove this commit with rebase? Or would you want me to revert this commit another way?

Reply:
As discussed in stand-up, will remove commit to be later implemented 👍